log_downloader_service.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import shutil
+from pathlib import Path
+from loguru import logger
+from git import Repo
+import wmi
+
+
+REPO_PATH = Path(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))) # TODO: change
```

Comment:
You can make better use of `pathlib` here:
```suggestion
REPO_PATH = Path(__file__).parent.parent # TODO: change
```
But why do you need specifically two directories up?

Reply:
I'm assuming the project is inside the robot code repo (I'll add it to the readme later).

Reply:
5b68894afa248ba89ac5d86abf60c2b41110afc6

Reply:
This is too rigid IMO. Why not just point to the directory for this event? I think it's the simplest way to address #7.